chains/v20/preConfigured/details/e143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e.json

```diff
@@ -0,0 +1,19 @@
+{
+    "chainId": "e143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e",
+    "name": "Westend",
```

Comment:
Westend (TESTNET)

Reply:
We have many testnets in preConfigured as we fetch networks from polkadotjs config, we can add this label only manually. Should we do this for all other nets or mark just Westend?